src/main/java/at/hannibal2/skyhanni/api/GetFromSackAPI.kt

```diff
@@ -222,12 +223,26 @@ object GetFromSackAPI {
     }
 
     @SubscribeEvent
-    fun onRepoReload(event: RepositoryReloadEvent) {
-        sackListInternalNames = event.getConstant<SacksJson>("Sacks").sackItems.toSet()
+    fun onNeuRepoReload(event: NeuRepositoryReloadEvent) {
+        val data = event.getConstant("sacks") ?: ErrorManager.skyHanniError("NEU sacks data is null.")
+        try {
+            val sacksData = ConfigManager.gson.fromJson<NeuSacksJson>(data).sacks
+            val uniqueSackItems = mutableSetOf<NEUInternalName>()
+
+            sacksData.values.forEach { sackInfo ->
+                sackInfo.contents.forEach { content ->
+                    uniqueSackItems.add(content)
+                }
+            }
+
+            sackListInternalNames = uniqueSackItems.map { it.asString() }.toSet()
+            sackListNames = uniqueSackItems.map { it.itemNameWithoutColor.uppercase() }.toSet()
```

Comment:
Why do we need uppercase here?

Reply:
ask thunderblade

Reply:
to make it case unsensitve. I made everything uppercase in the checking logic. Since InternalNames normlay are uppercase

Reply:
> Isnt it more intuitive to make the list neu internal name instead of string? We only need to make the user input to a NeuInternalName before comparing

I feel it would be better to have just one yes, but I also feel this is out of the scope for current pr as im unfamiliar with what most of this pr does so I dont think I should touch too much other code here

Reply:
The list is only used for comparing. And as a set of string it is a just a lookup from a hashmap not a search algorithm

Reply:
yeah agreed, out of scope of pr